docs/bis/guide/fundamentals/attribution-guidelines.md

```diff
@@ -0,0 +1,71 @@
+# Attribution: Guidelines
+
+BIS offers multiple ways to capture attributes of Entities being modeled. It is the job of the schema designer to choose the most appropiate strategy to follow in every case. This article aims to provide general guidelines for the choosing among those strategies.
+
+## Introduction
+
+Not every attribute of an Entity needs to be persisted into a BIS repository. Some attributes can be derived from persisted data, therefore saving them in a BIS repository turns derived attributes into effectively cached data that needs to be kept in-sync as the more fundamental attributes change. This situation effectively applies to iTwin Native apps that directly store their data in BIS repositories.
+
+## Persisted Attributes
+
+The following sections focus on attributes that are meant to be persisted in a BIS repository.
+
+### First-class Attributes
+
+An attribute of an Entity can be implemented as an [ECProperty](../../ec/ec-property.md) directly defined on the [Element-class](./data-classification.md#element-class) that represents it - also known as a `First-class property`. This is the strategy recommended to follow for attribution considered as *essential* according to the semantics and modeling perspective captured by the element-class modeling the associated Entity.
+
+Note, however, that there may be attributions whose applicability dependend on the state of a more general concept on a specific Entity. Such attributions shall be implemented via a class-hierarchy based on `Element-Aspects`(./elementaspect-fundamentals.md). If the overall concept is considered *essential* for the associated Entity, the target multiplicity of its associated `ElementOwnsUniqueAspect` or `ElementOwnsMultiAspect` relationship shall be set accordingly (i.e. 1..1 or 1..* respectively). See [Sub-Properties and ElementAspects](./elementaspect-fundamentals.md#sub-properties-and-elementaspects) for more information and an example of this situation.
+
+The default behavior of all UX controls in iTwin.js is to show First-class properties as attribution of their associated Element-class, unless they are selectively hidden by using the `HiddenProperty` or `HiddenClass` *Custom-Attributes*.
```

Comment:
I think it would be better to keep this doc neutral and just say they should be shown unless the hidden CAs are applied